packages/drivers/odsp-driver/src/test/odspError.spec.ts

```diff
@@ -283,4 +284,15 @@ describe("Odsp Error", () => {
         assert.strictEqual(error.redirectLocation, redirectLocation, "Site location should match");
         assert.strictEqual(error.statusCode, 404, "Status code should match");
     });
+
+    it("Sharepoint url should be redacted in the error", async () => {
+        try {
+            await mockFetchError(
+                async () => fetchAndParseAsJSONHelper("https://microsoft.sharepoint-df.com/siteUrl", {}),
+                new Error("Request to https://6c482541-f706-4168-9e58-8e35a9992f58.sharepoint.com failed"));
```

Comment:
Worth it to test more URL patterns that should get replaced? http, https, no subdomain, df and non-df.

Reply:
Redacting all urls now.